manage_print_jobs.go

```diff
@@ -45,15 +45,15 @@ func findPrinterToHandleFile(gcode GcodeFile) Print {
 	for {
 		updatePrinterStatus()
 		for i := range printerArray {
-			printer := printerArray[i]
+			printer := *printerArray[i]
```

Comment:
This actually dereferences the pointer before it gets passed to assignFileToPrinter()